client/my-sites/plans-v2/constants.ts

```diff
@@ -102,7 +102,7 @@ export const OPTION_PRODUCT_BACKUP: SelectorProduct = {
 	iconSlug: 'jetpack_backup_v2',
 	displayName: translate( 'Jetpack Backup' ),
 	tagline: '',
-	description: '',
+	description: 'Lorem ipsum',
```

Comment:
Since we are close to having a version that can be opened for internal testing I think it isn't a good idea to add placeholder text (we might forget to delete/change it). In https://wpcalypso.wordpress.com/devdocs/design we can showcase different states of a component. For instance, here is the `JetpackProductCard` component presented with different props: https://wpcalypso.wordpress.com/devdocs/design/jetpack-product-card. This file can help you add a new configuration for that component, if you think it would be of interest to add a new one, `client/components/jetpack/card/jetpack-product-card/docs/example.tsx`.

Reply:
Yes, good point, wouldn't want to forget to delete it. Ahh, cool!  It's like a UI component design system for all the calypso components!  Awesome! :)